optuna/structs.py

```diff
@@ -15,6 +16,9 @@
     from optuna.distributions import BaseDistribution  # NOQA
 
 
+StudyDirection = _study_direction.StudyDirection
```

Comment:
How about leaving a comment that this interface is deprecated? Although it's documented under `docs/`, the `docs/` directory is quite "far" from here.

Reply:
Thank you for your advice. As you suggested, the deprecation warning message in the `docs/` directory is far. I will note a deprecation comment "near" the definition of `structs.StudyDirection`.

Reply:
Thanks!